modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/ModifyTuple.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.sql.engine.schema;
+
+import org.apache.calcite.rel.core.TableModify;
+import org.apache.ignite.internal.schema.BinaryRow;
+
+/** POJO to store row and modify operation. */
+public class ModifyTuple {
+    private final BinaryRow row;
+
+    private final TableModify.Operation op;
```

Comment:
Let's introduce a internal enum `Type` (or something like this) instead of `TableModify.Operation`

Reply:
don`t understand the purpose of this change, also it can complicate further ports..

Reply:
Here we map a resulting row of particular ModifyNode operation to an operation of the table. Thus, it doesn't seem right to reuse operations enum of ModifyNode